sci-electronics/drahnr-oregano/drahnr-oregano-0.84.39.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python3_{4,5,6} )
+PYTHON_REQ_USE='threads(+)'
+
+inherit gnome2-utils python-single-r1 waf-utils xdg-utils
+
+DESCRIPTION="Application for the schematic capturing and simulation of electrical circuits"
+HOMEPAGE="https://github.com/drahnr/oregano"
+SRC_URI="https://github.com/drahnr/oregano/tarball/v${PV} -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+COMMIT="76560fa"
+
+S="${WORKDIR}/${PN}-${COMMIT}"
+
+PATCHES=(
+	"${FILESDIR}/${P}-fix-xdg.patch"
+	"${FILESDIR}/${P}-fix-gschema.patch"
+	)
+
+COMMON_DEP="dev-libs/glib:2
+	dev-libs/libxml2:2
+	x11-libs/goocanvas:2.0
+	x11-libs/gtk+:3
+	x11-libs/gtksourceview:3.0"
+
+DEPEND="${COMMON_DEP}
+	${PYTHON_DEPS}
+	virtual/pkgconfig"
+
+RDEPEND="${COMMON_DEP}
+	|| ( gnome-base/dconf gnome-base/gconf )
+	sci-electronics/electronics-menu"
+#	sci-electronics/ngspice
```

Comment:
Why are those dependencies commented out?

Reply:
It *may* use external soft, but not require it. I ask about it on IRC but no answer and comment it for keep and mind

Reply:
You're mentioning them in postinst. That's good enough (though you may want to list full package names there).

Reply:
I strongly recommend to use `ngspice` as a dependency, since the whole program is going to use ngspice by default. gnucap on the other hand should be optional since it is an alternate, less tested backend (not the porgram itself, just the wrapping code).